source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Removed the dark mode toggle option for Windows.
+
+MacOS
+^^^^^^^^^^^^^
+-
+
+All Platforms
+^^^^^^^^^^^^^
+- Added Swedish language to the spellchecker.
+- Added a setting to specify the default desktop app download location.
+- Updated the desktop app loading indicator.
+- Restored deeplinking.
+- Switched the dictionary to the one Electron uses to have the ability to fix spelling.
+- Added improvements to be consistent on the use of URL and URL libraries.
+- Removed the Finder functionality from the Desktop App.
+- Updated CSP for Desktop App to avoid warnings in the dev tools.
+- Removed the menu bar in the Settings window for Windows and Linux.
```

Comment:
Not sure what this is

Reply:
https://github.com/mattermost/desktop/pull/1530

Reply:
@Willyfrog Would you have details on how to describe this better? Was it a bug fix?

Reply:
@Willyfrog ^

Reply:
we can remove that from the list of changes, the actual change is that we now have the settings in a separate window. the PR mentions that we removed that because in linux and windows each window has its own menu window, while in mac it is a shared one